code/src/index.css

```diff
@@ -1,13 +1,304 @@
+  margin-left: 12rem;
+}
+
+.input-box{
+  margin-top: 8rem;
+  font-family: 'Alata', sans-serif;
+  display: flex;
+  flex-direction: column;
+  width: 290px;
+}
+
+.question-input{
+  font-size: 18pt;
+  margin-left: 20px;
+  color: #0a0a0a;
+  margin-top: -15%;
+  }
+
+.text-answer{ 
+  padding: 8px;
+  margin-top: 5%;
+  border: 0.16em solid #13341b;
+  border-radius: 2em;
+  font-family: 'Alata', sans-serif;
+  font-size: 14pt;
+}
+
+/*SecondQuestion*/
+
+.second-btn{
+  margin-top: 31rem;
+  margin-left: 12rem; 
+}
+
+.radio-form{
+  display: flex;
+  flex-direction: column;
+  margin-top: 2rem;
+  font-family: 'Alata', sans-serif;
+  font-size: 18pt;
+}
+
+.flavours{
+  display: block;
+  position: relative;
+  padding-left: 35px;
+  margin-bottom: 12px;
+  cursor: pointer;
+  font-size: 22px;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
+}
+
+/*ThirdQuestion*/
+
+.third-btn{
+  margin-top: 31rem;
+  margin-left: 12rem; 
+}
+
+.list-question{
+  font-family: 'Alata', sans-serif;
+    width: 80%;
+    margin-left: 10%;
+    margin-top: 2rem;
+    text-align: center;
+    font-size: 18pt;
+}
+
+.drop-down{
+  font-family: 'Alata', sans-serif;
+  font-size: 14;
+    padding: 12px;
+    border-radius: 30px;
+    margin: 10px;
+    width: 250px;
+}
+
+/*Summary*/
+
+.summary-text{
+  font-family: 'Alata', sans-serif;
+  text-align: center;
+  color:#c13e29;
+  font-size: 16pt;
+  margin-top: 4rem;
+  padding:1rem;
+  background-color: #e1934b;
+  border-radius: 20px;
+  border: 0.1em solid #c13e29;
+  opacity: 0.9;
+  box-shadow:
+  0 0 10px #fff,
+  0 0 21px #fff,
+  0 0 42px rgb(255, 0, 234),
+  0 0 82px rgb(0, 255, 115),
+  0 0 102px rgb(157, 255, 0),
+  0 0 151px rgb(255, 238, 0);
+}
+
+.summary{
+  font-family: 'Alata', sans-serif;
+  text-align: left;
+  color: #13341b;
+  width: 75%;
+  background-color: #a1d854;
+  font-size: 14pt;
+  margin-top: 3rem;
+  padding: 1rem;
+  border-radius: 17px;
+  margin-left: 11px;
+  opacity: 0.9;
+  box-shadow: 0 0 10px #fff, 0 0 21px #fff, 0 0 42px rgb(255 0 234), 0 0 82px rgb(0 255 115), 0 0 102px rgb(157 255 0), 0 0 151px rgb(255 238 0);
+}
+
+.end-text{
+  font-family: 'Bangers', cursive;
+    font-size: 70px;
+    color: #c43248;
+    opacity: 0.9;
+    margin-top: 2rem;
+    margin-left: 3rem;
+    transform: rotate(-10deg);
+  text-shadow: 0 0 10px #fff, 0 0 21px #fff, 0 0 82px rgb(0 255 115), 0 0 151px rgb(255 238 0);
+}
+
+  /*Footer*/
+  
+  footer{
+    background-color: rgb(185 71 85 / 87%);
+    height: 10vh;
+    display: flex;
+    flex-direction: column;
+    margin-top: auto;
+    text-align:left;
+  }
+  
+  .creator{
+    font-family:'Alata', sans-serif;
+    font-size:xx-small;
+    margin:1rem;
+  }
+
+  /*MediaQueries*/
+
+  @media screen and (min-width: 600px) {
+    .intro-container{
+      width:75%;
+      min-height: 40%;
+      margin-left: auto;
+    }
+    .start-text{
+      transform: rotate(-5deg);
```

Comment:
When u are useing media queries a tip is use the toggle device so u can se how the page moves, on some parts u have the button and dropdown menus on top of each other at some moments.